workshops/agctools2022/statistical-inference/exclusion/src/exclusion/visualize.py

```diff
@@ -125,7 +125,3 @@ def plot_contour(ax, results, **kwargs):
         )
 
     return ax
-
-
-if __name__ == "__main__":
-    main()
```

Comment:
I guess this gets removed because `main` isn't defined? Not sure either what the purpose of it was. @kratsg / @matthewfeickert do you know?